server/conf/timely-standalone.yml

```diff
@@ -0,0 +1,47 @@
+timely:
+  ip: 0.0.0.0
+  port:
+    put: 54321
+    query: 54322
+    websocket: 54323
+  instance-name: TimelyStandalone
```

Comment:
I think in order to have one Configuration class that can be passed around, the accumulo group will need to be nested in the timely group. If we were autowiring everything, then you could just inject the configuration class you need. I'll move the accumulo group inside timely and go from there.